apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-user-fields.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * WP_REST_Help_Center_User_Fields file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * Class WP_REST_Help_Center_User_Fields.
+ */
+class WP_REST_Help_Center_User_Fields extends \WP_REST_Controller {
+	/**
+	 * WP_REST_Help_Center_User_Fields constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'help-center';
+		$this->rest_base = '/zendesk/user-fields';
+	}
+
+	/**
+	 * Register available routes.
+	 */
+	public function register_rest_route() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base,
+			array(
+				'methods'             => \WP_REST_Server::CREATABLE,
+				'callback'            => array( $this, 'update_user_fields' ),
+				'permission_callback' => array( $this, 'permission_callback' ),
```

Comment:
```suggestion
				'permission_callback' => array( $this, 'is_user_logged_in' ),
```

Reply:
Actually I think it would be:

```suggestion
				'permission_callback' => 'is_user_logged_in',
```

And this would alleviate from having to add the `permission_callback` below.

Reply:
I like the shorter syntax (and you can see the check inline), but I'd prefer to keep it as-is:
 - It's more consistent with other endpoints.
 - It's easier to extend the permission check - the diff would be smaller.